### PR TITLE
Updated Yarn installation instructions in README.md

### PR DESCRIPTION
**Pull request by jkowalcz**

### CODE REVIEW

**Review by PharkMillups — COMMENTED** [review]
I think this looks good. @kent-williams Can you give it a quick look?

**Comment by kent-williams** [comment]
@jkowalcz  This is turning into an installation guide. I would prefer if we kept the readme lean with the essentials.  We should add a guide in the docs for these kinds of instructions. Could you please remove all the additions except the update to the yarn url on line 9.

**Comment by jkowalcz** [comment]
> @jkowalcz This is turning into an installation guide. I would prefer if we kept the readme lean with the essentials. We should add a guide in the docs for these kinds of instructions. Could you please remove all the additions except the update to the yarn url on line 9.

@kent-williams  No I completely understand, I'm looking to jump start people into the full development environment, as its not as simple as it is put out in the README file as it currently sits.  

How about I move it to it's own file? YARN-INSTALL.md or something? Then I can just reference that in the README.md and Yarn file can change and be updated.

**Comment by jkowalcz** [comment]
> YARN-INSTALL.md 

CONTRIBUTING.md is probably a better spot for it all maybe?

**Comment by kent-williams** [comment]
I'd prefer we didn't add any more files to the repo or touch contributing(this is standard across all our repos). The actual docs site would be perfect for this, we can link to it from the readme. Perhaps we could put it under https://docs.helium.com/open-source/docs/installation.

**Comment by jkowalcz** [comment]
> I'd prefer we didn't add any more files to the repo or touch contributing(this is standard across all our repos). The actual docs site would be perfect for this, we can link to it from the readme. Perhaps we could put it under https://docs.helium.com/open-source/docs/installation.

@kent-williams Oh Yeah that works too!  I like it. Let me rework it.

**Comment by jkowalcz** [comment]
> @kent-williams Oh Yeah that works too! I like it. Let me rework it.

@kent-williams  Check this out, looks much better.

File: docs/open-source/installation.mdx [hunk 1]
```diff
@@ -0,0 +1,121 @@
+---
+id: installation
+hide_title: true
```
Comment (kent-williams) [inline docs/open-source/installation.mdx]: Remove this so that the doc title is shown at the top.

File: docs/open-source/installation.mdx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+sidebar_label: Installation
+slug: /open-source/installation
+---
+
+# Open source Installation details
```
Comment (kent-williams) [inline docs/open-source/installation.mdx]: Let's make sure it's clear that this guide is for the Docs Website Installation.

File: docs/open-source/installation.mdx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+i ｢wds｣: 404s will fallback to /index.html
+
+* Client ... done (99%) plugins
+ WebpackDevMiddleware
+```
+
+This command starts a local development server and open up a browser window. Most changes are reflected live without having to restart the server.
+
+:::warning
+You can not use `git add .` moving forward at the root of the repository. There are yarn files in the root of the project that should not be committed
```
Comment (kent-williams) [inline docs/open-source/installation.mdx]: You should never be using `gid add .` unless you are starting a fresh repository, perhaps we can remove this.

Comment (jkowalcz) [reply docs/open-source/installation.mdx]: I added it cause I'm a lazy git developer and use it :) I can remove it but it's a warning to other lazy people.

Comment (kent-williams) [reply docs/open-source/installation.mdx]: Yeah we probably don't want to be advertising poor git form.

**Comment by kent-williams** [comment]
Looks good, thanks for doing all those changes!

**Review by kent-williams — APPROVED** [review]

**Pull request merged**